blockchain/treasury.go

```diff
@@ -1089,7 +1089,7 @@ func (b *BlockChain) checkTSpendHasVotes(prevNode *blockNode, tspend *dcrutil.Tx
 			maxRemainingVotes)
 	}
 
-	trsyLog.Infof("TSpend %v passed with: yes %v no %v quorum %v "+
+	trsyLog.Debugf("TSpend %v passed with: yes %v no %v quorum %v "+
```

Comment:
ACK, but I did like to see this:

> 2021-12-28 12:30:20.869 [INF] TRSY: TSpend f57f2f35abcfc58cb8d45da6315982610203c1335ee0d68b36240f22c24a557a passed with: yes 6269 no 0 quorum 3456 required 5489

I'll probably start setting the `TRSY` subsystem to debug.

Reply:
Yeah, this should definitely be debug level.  It's also the only thing that would be shown at debug, so using debug logging for it would not cause a bunch of extra logging.